library/core/src/cell.rs

```diff
@@ -658,7 +663,12 @@ impl<T> RefCell<T> {
     #[rustc_const_stable(feature = "const_refcell_new", since = "1.32.0")]
     #[inline]
     pub const fn new(value: T) -> RefCell<T> {
-        RefCell { value: UnsafeCell::new(value), borrow: Cell::new(UNUSED) }
+        RefCell {
+            value: UnsafeCell::new(value),
+            borrow: Cell::new(UNUSED),
+            #[cfg(feature = "debug_refcell")]
+            first_caller: Cell::new(None),
```

Comment:
The name `first_caller` isn't very obvious to me. How about something like `borrowed_by` or `borrowed_at` or `borrowed_first_at`?